tests/field/datePick.test.tsx

```diff
@@ -164,4 +164,40 @@ describe('Field', () => {
       '<div><div><div>2016-11-22 15:22:44</div><div>2016-11-23 15:22:44</div></div></div>',
     );
   });
+
+  it(`📅  DatePicker support format is Array`, async () => {
+    const fn = jest.fn();
+    const html = render(
+      <Field
+        mode="read"
+        fieldProps={{
+          format: ['YYYY-MM-DD', 'YYYYMMDD'],
+        }}
+        onChange={fn}
+        text={dayjs()}
+        light
+        valueType="date"
+      />,
+    );
+
+    expect(html.baseElement.innerHTML).toBe('<div><div>2016-11-22</div></div>');
+  });
+
+  it(`📅  DatePicker support format is Array`, async () => {
```

Comment:
There's no need to write two test cases, you can combine them into one test case, and test both cases in the same test case

Reply:
got it,thanks